crates/ruff_workspace/src/resolver.rs

```diff
@@ -699,7 +732,7 @@ pub fn python_file_at_path(
         for ancestor in path.ancestors() {
             if let Some(pyproject) = settings_toml(ancestor)? {
                 let (root, settings) =
-                    resolve_scoped_settings(&pyproject, Relativity::Parent, transformer)?;
+                    resolve_scoped_settings(&pyproject, Relativity::Parent, transformer, None)?;
```

Comment:
It's not clear to me why using `None` here is correct (or when using `None` is correct in general). Can you tell me a bit more about it?